src/rules/banRule.ts

```diff
@@ -133,10 +137,14 @@ function parseOptions(args: Array<string | string[] | OptionsInput>): Options {
                 case 0:
                     break;
                 case 1:
-                    functions.push({name: arg.name[0], message: arg.message});
+                    functions.push({ name: arg.name[0], message: arg.message });
                     break;
                 default:
-                    methods.push({name: arg.name[arg.name.length - 1], object: arg.name.slice(0, -1), message: arg.message});
+                    methods.push({
+                        name: arg.name[arg.name.length - 1],
+                        object: arg.name.slice(0, -1),
+                        message: arg.message
```

Comment:
So just to confirm, if you move `message` to the beginning of this object, the commit hook moves it back to the end?

Reply:
It doesn't and I see why. The formatter on my commit changes the line

```typescript
methods.push({name: arg.name[arg.name.length - 1], object: arg.name.slice(0, -1), message: arg.message});
```

into the snippet you posted.

If I then make another commit, the formatter runs again and move `message` up into the correct sorting position. It looks like the formatter can't make both edits in one commit.

Reply:
What I can do is squash this second commit down and try pushing up again.

Reply:
The formatter is not able to correct this file no matter how many times it runs.

Reply:
I manually moved the `message` line up and CI passes now. I guess I don't understand how much the formatter is supposed to change and falsely believed it should correct the sort order.

Reply:
Awesome. The autoformatter is a bit of a pain, but hopefully will let us work towards a codebase that is nicer without ruining everyones open PRs.